packages/react-scripts/config/webpack.config.js

```diff
@@ -170,6 +170,7 @@ module.exports = function(webpackEnv) {
       // We inferred the "public path" (such as / or /my-project) from homepage.
       // We use "/" in development.
       publicPath: publicPath,
+      globalObject: 'this',
```

Comment:
This is required due to what seems to be a bug/regression in webpack 4. More details can be found here: https://github.com/webpack/webpack/issues/6642

Reply:
Hello @iansu , can i ask a question . Why **globalObject** value is **this** , not **self** ?

Reply:
+1

Reply:
@ri7nz -- as far as I can tell, the eslint config inside ``create-react-app`` bugs out on ``self`` keyword saying it's invalid, and hence the empty production build when using worker-loader yourself.